code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -794,21 +810,29 @@
 // Putting these at the bottom so they don't clutter the list up. -Cheridan
 // *************************************
 
-/obj/item/weapon/reagent_containers/food/snacks/grown/bluespacetomato
-	name = "tomato" //"blue-space" is applied on new(), provided it's teleporting trait hasn't been removed
-	desc = "Its juices lubricate so well, you might slip through space-time."
-	icon_state = "bluespacetomato"
-	potency = 20
-	origin_tech = "bluespace = 3"
-	filling_color = "#91F8FF"
-	plantname = "bluespacetomato"
+/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac
+	plantname = "vaporsac"
+	name = "vapor sac fruit"
+	desc = "A thin organic film bearing seeds, held slightly aloft by internal gasses and a reservoir of chemicals."
+	icon_state = "vaporsac"
+	filling_color = "#FFFFFF"
 
-/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiavulgaris/cruciatus
-	plantname = "ambrosiacruciatus"
-	name = "ambrosia vulgaris branch"
-	desc = "This is a plant containing various healing chemicals."
-	icon_state = "ambrosiavulgaris"
-	potency = 10
+/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac/attack(mob/living/M, mob/user, def_zone, eat_override = 0)
+	pop(user)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac/attack_animal(mob/M)
+	pop(M)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac/attackby(obj/item/weapon/W, mob/user)
+	pop(user)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/vaporsac/proc/pop(mob/popper)
+	if(popper)
+		popper.visible_message("<span class='warning'>[popper] pops the \the [src]!</span>","<span class='warning'>You pop \the [src]!</span>")
+	for(var/mob/living/carbon/C in view(1))
+		C.Weaken(5)
```

Comment:
Yeah, but there's nothing the person who pops it can do to avoid the stun either. It's also half as much as a flash. Vox have access to flashes and flashbangs from their secvendor anyway.

Now, in theory if someone had their mind put into a simple mob (not carbon) and dragged one around with them, they would be able to stun someone one time without getting stunned themselves. But that seems a bit niche, don't you think?